docs/operators/how-to/configure-store.md

```diff
@@ -1,56 +1,57 @@
+
+```shell
+wakunode2 --store=false
 ```
 
-Note that this only mounts the `store` protocol,
-meaning your node will indicate to other peers that it supports `store`.
-It does not yet allow your node to either retrieve historical messages as a client
-or store and serve historical messages itself.
+This option controls the mounting of the Waku Store protocol, meaning that your node will indicate to other peers that it supports the Waku store protocol.
 
-## Configuring a store client
+## Configuring the node as a waku store client
 
-Ensure that `store` is enabled (this is `true` by default) and provide at least one store service node address with the `--storenode` CLI option.
+Provide at least one store service node address with the `--storenode` option. This option is independent of the `--store` option i.e., one node can act as a waku store client without mounting the Waku Store protocol.
 
-See the following example, using the peer at `/dns4/node-01.ac-cn-hongkong-c.wakuv2.test.statusim.net/tcp/30303/p2p/16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm` as store service node.
+For example, to use the peer at `/dns4/node-01.ac-cn-hongkong-c.wakuv2.test.statusim.net/tcp/30303/p2p/16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm` as the waku store service node:
 
-```sh
-./build/wakunode2 \
-  --store:true \
-  --storenode:/dns4/node-01.ac-cn-hongkong-c.wakuv2.test.statusim.net/tcp/30303/p2p/16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm
+```shell
+wakunode2 \
+  --storenode=/dns4/node-01.ac-cn-hongkong-c.wakuv2.test.statusim.net/tcp/30303/p2p/16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm
 ```
 
 Your node can now send queries to retrieve historical messages
-from the configured store service node.
-One way to trigger such queries is asking your node for historical messages using the [Waku v2 JSON RPC API](https://rfc.vac.dev/spec/16/).
-
-## Configuring a store service node
-
-To store historical messages on your node which can be served to store clients the `--persist-messages` CLI option must be enabled.
-By default a node would store up to the latest `50 000` messages.
-This is configurable using the `--store-capacity` option.
-A node that has a `--db-path` set will backup historical messages to a local database at the DB path
-and persist these messages even after a restart.
-
-```sh
-./build/wakunode2 \
-  --store:true \
-  --persist-messages:true \
-  --db-path:/mnt/nwaku/data/db1/ \
-  --store-capacity:150000
+from the configured store service node. One way to trigger such queries is asking your node for historical messages using the [Waku v2 JSON RPC API](https://rfc.vac.dev/spec/16/).
+
+## Configuring the node as a store service node
+
+If the waku store node is enabled (the `--store` option is set to `true`) the node will store historical messages and will be able to serve those messages to the waku store clients.
```

Comment:
> Does that mean storing messages is default? (Because `--store=true` is default)

Yes, that's what I meant with _"the node will store historical messages"_.

> I'd make it explicit above that you need to set `--store=false` and set `--storenode` to setup a client only store-node.

This is not the case. Both things (server and client) are independent. You can have both on, both off, or one on and the other off.

This is what I meant with this: https://github.com/status-im/nwaku/pull/1359/files?diff=split&w=0#diff-a2d58beafcccb9e2b626c20cb0e31383f1107c520e7a427e20d2896763f71eb7R16

Reply:
True, but it may be worth spelling out in the client section that to configure a client _only_ `store` must explicitly be set to `false`. This is clearly implied, of course, but we should assume that operators will just skip to the section they're interested in and may be surprised that following the client guidelines lead to a node that's also persisting messages.

Not part of this conversation, but wondering if shouldn't rather make the default for `store` `false`. Otherwise many operators upgrading after the release will suddenly start persisting messages. We can warn them, of course, but I think the sensible default for a nwaku node should just be a `relay` node.